shared/attestationutil/attestation_utils.go

```diff
@@ -65,14 +68,17 @@ func ConvertToIndexed(ctx context.Context, attestation *ethpb.Attestation, commi
 //    """
 //    committee = get_beacon_committee(state, data.slot, data.index)
 //    return set(index for i, index in enumerate(committee) if bits[i])
-func AttestingIndices(bf bitfield.Bitfield, committee []uint64) []uint64 {
+func AttestingIndices(bf bitfield.Bitfield, committee []uint64) ([]uint64, error) {
+	if bf.Len() > uint64(len(committee)) {
```

Comment:
This should return an error if they are not equal.

Reply:
True